components/resourceDetails/resourceSharing/addAgentButton/agentPickerModal/__snapshots__/index.test.jsx.snap

```diff
@@ -9,7 +9,358 @@ exports[`AgentPickerModal renders a table with tabs, searchbox and an empty stat
     <div
       class="PodBrowser-title"
     >
-      Add Editors for null
+      Edit Editors for null
```

Comment:
Minor nitpick - I think your test is missing something, since we see `null`?

Reply:
The dataset name is fetched from  useContext(DatasetContext)` and I thought it wasn't worth the trouble of trying to mock a context.

Reply:
Fair enought